features/environment.py

```diff
@@ -465,8 +466,24 @@ def after_scenario(context, scenario):
                 logging.warning("Failed to collect coverage")
 
 
+def _get_apparmor_logs_from_host():
+    # get apparmor DENIED messages from the host
+    with open("/var/log/syslog", "r") as syslog_fd:
```

Comment:
If `auditd` is running, apparmor DENIED messages will go to `/var/log/audit/audit.log`, not syslog. This looks like a test framework, so it's probably fine.

Reply:
Correct. I'm using this to easily get access to apparmor logs from the github runners. I don't know yet if I will leave it in here, or change it.